build/jakarta/migrate.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+script_dir_path=$(cd "$(dirname "${BASH_SOURCE[0]}")" || exit; pwd -P)
+
+mvn_cmd="mvn ${BUILD_MVN_OPTS:-}"
+optaplanner_file="${script_dir_path}/optaplanner-quarkus3.yaml"
```

Comment:
TODO: move this to `optaplanner-migration` once the Quarkus 3 recipe is available as a dependency.